### PR TITLE
Expand intro in README

### PR DESCRIPTION
**Pull request by turbodog**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,15 @@
-# vRO Starter Content Pack
+# vRealize Plugin Starter Content Pack
```
Comment (bmjen) [inline README.md]: Shouldn't this be `vRealize Orchestrator Plug-in`?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
 
-## Introduction
+## Introduction
```
Comment (bmjen) [inline README.md]: trailing space?

File: README.md (hunk 1 above)
Comment (turbodog) [reply README.md]: vRealize Automation is the preferred term as a catchall for VRO and VRA

**Comment by bmjen** [comment]
@turbodog commits needs squash before merge.

**Pull request merged**